samplekotlin/src/main/java/com/example/chattutorial/ImgurAttachmentFactory.kt

```diff
@@ -47,12 +47,12 @@ class ImgurAttachmentFactory : AttachmentFactory {
             binding.ivMediaThumb.apply {
                 shapeAppearanceModel = shapeAppearanceModel
                     .toBuilder()
-                    .setAllCornerSizes(resources.getDimension(R.dimen.stream_ui_selected_attachment_corner_radius))
+                    .setAllCornerSizes(resources.getDimension(io.getstream.chat.android.ui.R.dimen.stream_ui_selected_attachment_corner_radius))
```

Comment:
Not using fully classified names seems to throw the IDE off when copy pasting from our tutorial or even another IDE where the import is working.

Classifying it fully fixes that and should result in the users a smoother experience